docker/eradiate-kernel-builder/Dockerfile

```diff
@@ -22,10 +22,8 @@ ENV CXX=clang++-9
 COPY . /sources/eradiate
 WORKDIR /sources/eradiate
 RUN make conda-init
-RUN git submodule update --init --recursive
 
-RUN mkdir -p /build/eradiate-kernel                                                                                       \
-    && cd /build/eradiate-kernel                                                                                          \
-    && cmake -GNinja -DPYTHON_EXECUTABLE=$(python3 -c "import sys; print(sys.executable)") /sources/eradiate/ext/mitsuba2 \
-    && ninja                                                                                                              \
-    && tar -cvf eradiate-kernel-dist_${ERADIATE_KERNEL_VERSION}.tar ./dist
+WORKDIR /build/eradiate-kernel
+RUN cmake -GNinja -DPYTHON_EXECUTABLE=$(python3 -c "import sys; print(sys.executable)") /sources/eradiate/ext/mitsuba2
+RUN ninja
```

Comment:
The new build process requires you to use the CMake presets to compile the appropriate variants and disable Embree: https://eradiate.readthedocs.io/en/next/rst/getting_started/install.html#compiling-the-kernel

Also, you no longer need to pass the `PYTHON_EXECUTABLE` variable, pybind11 is smarter than before :)

Reply:
This should be fixed now